Source/controls/controller_motion.cpp

```diff
@@ -149,30 +149,32 @@ bool IsControllerMotion(const SDL_Event &event)
 	}
 #endif
 
+#if defined(JOY_AXIS_LEFTX) || defined(JOY_AXIS_LEFTY) || defined(JOY_AXIS_RIGHTX) || defined(JOY_AXIS_RIGHTY)
 	if (event.type == SDL_JOYAXISMOTION) {
 		switch (event.jaxis.axis) {
 #ifdef JOY_AXIS_LEFTX
 		case JOY_AXIS_LEFTX:
 			return true;
 #endif
-#ifdef JOY_AXIS_LEFTX
+#ifdef JOY_AXIS_LEFTY
 		case JOY_AXIS_LEFTY:
 			return true;
 #endif
-#ifdef JOY_AXIS_LEFTX
+#ifdef JOY_AXIS_RIGHTX
 		case JOY_AXIS_RIGHTX:
 			return true;
 #endif
-#ifdef JOY_AXIS_LEFTX
+#ifdef JOY_AXIS_RIGHTY
 		case JOY_AXIS_RIGHTY:
 			return true;
 #endif
 		default:
 			return false;
 		}
 	}
-
+#else
 	return false;

```

Comment:
The `return false;` is always needed because the `event.type` might not be `SDL_JOYAXISMOTION`.

Reply:
Good point. Fixed. 🙂